### ISSUE TITLE
STDLIB_DIR and SP_DIR not set in build environment for conda-build 2.1.5

### ISSUE THREAD

**Issue by tomashek** [conda/conda-build#1760]
While building cpython 3.5.2 on macOS 10.11, I found that neither SP_DIR nor STDLIB_DIR are being set in the environment. Both are set when building the same recipe using conda-build 1.21.11.

Reproducer:
add "env > $HOME/buildenv.txt" to the top of the build script. buildenv.txt contains:
STDLIB_DIR=
SP_DIR=

**Comment by AndresGuzman-Ballen** [comment]
@msarahan While trying to build python with a recipe of ours using conda-build 2.1.10, the STDLIB_DIR environment variable is not found. I did a simple 'mkdir %STDLIB_DIR%\hmm' and only found it on my C drive. Was this commit supposed to address that problem? I saw the conda-build/environ.py file was modified recently but they don't seem to include these kinds of changes. Should I file a ticket under conda-build's issues?
